submarine-test/test-k8s/src/test/java/org/apache/submarine/rest/NotebookRestApiIT.java

```diff
@@ -188,6 +189,7 @@ public void testListNotebooksWithUserId() throws Exception {
     Assert.assertEquals(ENV_NAME, getEnvironment.getEnvironmentSpec().getName());
 
     // create notebook instances
+    Thread.sleep(15000);
```

Comment:
Could you add some comments for `Thread.sleep(15000);`?  Don't know why we should use it.

Reply:
In this test we use different ways to create notebook with the same name and delete it in very short time, but persistent volume (pv) can not be deleted quickly. When previous pv haven't been deleted completely and we create new pv with the same name, the error will occur, so I choose to add Thread.sleep. I will add some comments in the file.